supplemental_notebooks/2020-06-26_supplemental-notes.Rmd

```diff
@@ -0,0 +1,154 @@
+```
+
+An advantage of organizing the directories this way is that it makes it easy to adapt the tximport processing steps for the gastric cancer dataset to process the neuroblastoma cell line by changing the last argument to `file.path()`.
+
+```{r eval=FALSE}
+# The path to the quant files for the gastric cancer dataset
+file.path("data", "quant", "gastric_cancer")
+# The path to the quant files for the neuroblastoma cell line dataset
+file.path("data", "quant", "NB_cell_line")
+```
+
+For data types that have files on a _per sample basis_ (FASTQ, quant), rather than for an entire dataset/experiment, the files for a sample are in a directory named based on a unique identifier for that sample.
+
+Here's an example from the neuroblastoma cell line quant files:
+
+```{r}
+dir(file.path("data", "quant", "NB_cell_line"))
+```
+
+## Filtering DESeq2 DGE results for downstream analysis
+
+During the main session on differential expression analysis, we sorted results based on the Wald statistic. The Wald test is the hypothesis test for comparing two groups; it also produces the p-values in the results that are then corrected for multiple hypothesis testing via Benjamini-Hochberg (BH) adjustment (default settings). 
+
+> BH-adjusted p-values are often referred to as false discovery rates or FDR.
+> For an explainer on FDR values, check out [this excellent video from StatQuest](https://www.youtube.com/watch?v=K8LQSvtjcEo).
+
+In practice, we're often interested in genes with low FDR values and the highest magnitude of log2 fold changes (LFC).
+(This is the rationale behind volcano plots like the one we made in the Intro to R and the Tidyverse module.)
+
+### Load in results from neuroblastoma cell line 
+
+Yesterday, we saved the object returned by `DESeq2::DESeq()` as an RDS file.
+
+```{r message=FALSE}
+deseq_object_file <- file.path("results", "NB_cell_line_DESeq_amplified_v_nonamplified.RDS")
+deseq_object <- readr::read_rds(deseq_object_file)
+```
+
+We need to load in the `DESeq2` library and we'll use the `%>%` operator later.
+
+```{r}
+library(DESeq2)
+`%>%` <- dplyr::`%>%`
+```
+
+### Shrunken log2 foldchanges
+
+There's an additional step we can take to get more accurate log2 fold change estimates to use for visualization or ranking.
+
+> We **highly recommend** taking a look at the resources we adapted this section from!
+>
+> * [Harvard Chan Bioinformatics Core (HBC). _DGE Analysis: Pairwise Comparisons (Wald Test)._](https://hbctraining.github.io/DGE_workshop_salmon/lessons/05_DGE_DESeq2_analysis2.html)
+> * [Love, Anders, and Huber. _Analyzing RNA-seq data with DESeq2._](https://bioconductor.org/packages/devel/bioc/vignettes/DESeq2/inst/doc/DESeq2.html#log-fold-change-shrinkage-for-visualization-and-ranking)
+
+```{r}
+# Look at the contrasts available
+resultsNames(deseq_object)
+```
+
+We compared the _MYCN_ amplified cell lines to the non-amplified cell lines.
+Shrink the LFC for more accurate estimates.
```

Comment:
Do you want to say anything about more about shrinkage? The link may be enough, but maybe a sentence in case they have never heard the term?